Source/Fuse.Reactive.JavaScript/Tests/DITest.uno

```diff
@@ -0,0 +1,31 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+using Uno.Testing;
+
+using Fuse.Controls;
+using Fuse.Navigation;
+using FuseTest;
+
+namespace Fuse.Reactive.Test
+{
+	public class DITest : TestBase
+	{
+		[Test]
+		public void DIBasics()
+        {
```

Comment:
The indentation went wacky here (infiltration of spaces maybe).

Consider `using (var root = TestRootPanel.CreateWithChild(e))` instead. This form will include cleanup tests.